src/conmon.c

```diff
@@ -84,6 +84,16 @@ int main(int argc, char *argv[])
 		exit(0);
 	}
 
+	char *notify_socket_path = getenv("NOTIFY_SOCKET");
```

Comment:
is there a reason to expose this via env vs cli?

Reply:
Just that it's how the notify socket is usually passed from systemd et al - but podman is always our parent, so I'm fine with whatever.

Maybe --notify-socket-host and --notify-socket-destdir or something - that can get rid of the hardcoded path of /run/notify/notify.sock in the container, and instead have it be (notify-socket-destdir)/notify.sock in the container.

Reply:
yes I like that idea. then it can be exposed via cli on podman's side if the user wants to change it.

Reply:
could we include the commit `52546c919be261e74847981b4db1227537c47345` into the commit that first introduced the change?

Also, I think we need a way to disable the notify socket to be handled by conmon, there are cases when it is helpful to use the handling in the OCI runtime.  For example when the user needs `$RUNTIME start $CTR` to wait for the container to be ready.